addons/website_sale/static/src/plugins/checkout_page_option.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="website_sale.checkoutPageOption">
+    <BuilderRow label.translate="Extra Step">
```

Comment:
if you want, you can wrap all of this in a BuilderContext to define the action, so you don't have to redefine it each time you use it